MAINTAINERS.yml

```diff
@@ -1031,6 +1031,21 @@ Documentation:
     labels:
         - "area: Wifi"
 
+"Drivers: Xen":
+    status: maintained
+    maintainers:
+        - povergoing
+    collaborators:
+        - SgrrZhf
```

Comment:
Should add others highly involved in this area as collaborators:

* @lorc (initial contributor)
* @firscity (had many related contributions)

unless, of course, they do not want to be added as collaborators.

Reply:
I also have no objections.

Reply:
Sure, I will

Reply:
@luca-fancellu would like to be the collaborator, he has a relevant strong experience in Xen. I will add him too.

Reply:
@stephanosio @carlocaione would you like to be the collaborators?

Reply:
unfortunately I'm not familiar with XEN, so my contribution could be not that valuable. But thanks for the thought ;)

Reply:
> unfortunately I'm not familiar with XEN, so my contribution could be not that valuable. But thanks for the thought ;)

Np, welcome to review. Also, we may bother you if there is something we are unsure about, hah.

Reply:
Thank you @povergoing